metrics.cpp

```diff
@@ -0,0 +1,139 @@
+/* This is an example how to add additional evaluation metrics to GraphLab Create using the SDK.
+In this example, we developed seven evaluation metrics function for binary classifier. 
```

Comment:
I feel  like you need to have a python version as well so people can get started more easily. You can have 2 how to's with SDK and python (same content)